curriculum/challenges/_meta/learn-the-css-box-model-by-building-a-rothko-painting/meta.json

```diff
@@ -67,129 +67,125 @@
       "id": "60a3e3396c7b40068ad69977",
       "title": "Step 14"
     },
-    {
-      "id": "60a3e3396c7b40068ad69978",
```

Comment:
It looks like you made changes to the wrong file here.

This is for a different project.

I would suggest reverting your last commit and then removing the duplicate step in the flexbox project

Reply:
Ohh ok

Reply:
> It looks like you made changes to the wrong file here.
> 
> This is for a different project.
> 
> I would suggest reverting your last commit and then removing the duplicate step in the flexbox project

Made the changes @jdwilkin4

Reply:
The duplicate issue for the flexbox project has been resolved but you will still need to revert this file back to its original version because you have altered steps for another project. 

Basically, this PR should not make any changes to the meta.json file for the css box model project because this PR should only be focused on the flexbox project

Reply:
Ooh ok. Uhhh, which ones should revert again?

Reply:
I think the easiest way would be to revert back to this commit here 587279bf4821ede456d54047d144eb910f12903e

Then you can make another commit to remove the duplicate from the correct project like you did in this last commit here 
https://github.com/freeCodeCamp/freeCodeCamp/pull/50795/commits/0f21fcbc566a233418a302923892c6a1d064a1b3

Reply:
So...revert commit 587279b?

Reply:
yes, you should go back to that commit.
The duplicate will only temporarily appear again because you can always push up another commit to correct the issue in the correct project.

Remember, that this repo uses squash and merge so it doesn't matter how many commits you have in the PR

Reply:
Ah ok